README.md

```diff
@@ -19,43 +19,79 @@ operator. In this lesson, we're going to learn about all three.
 Let's say you work for an e-commerce site, and you're prepping for your
 post-holiday sales. You're working on some code for your website and you need to
 set a discount of 25% across the board for everything that you sell on the
-website. We have a function that takes in an `item` as a price in dollars and a
-`discount` as a percentage, and returns the total amount due.
+website. 
+
+We have a function that takes in an `itemPrice` as a price in dollars and a
+`discount` as a percentage, and returns the total amount due:
 
 ```js
-function sellStuff(item, discount){
-    return item - (item * discount)
+function discountedPrice(itemPrice){
+    return itemPrice - (itemPrice * 0.25)
 }
 ```
 
-Since normally our discounts are on a per-item basis, this code functions well.
-But it could be a tiny bit more efficient now that we're going to be setting a
-25% off across the board discount. We can set a default 25% off discount by
-making a tiny change in our code like this:
+But it seems odd to hard-code the discount to `0.25`. We want to make this
```

Comment:
Instead of using the word "odd" here, I would suggest using something like "inefficient," to make it clear exactly why we prefer to not hard-code.